qiita_db/job.py

```diff
@@ -92,6 +93,10 @@ def exists(cls, datatype, command, options, analysis,
             Options for the command in the format {option: value}
         analysis : Analysis object
             The analysis the job will be attached to on creation
+        input_file_reference_id : int
```

Comment:
We've already standardized on passing in the string though, so we should stick with previous convention and use a standardized way of calling things instead of a mixed string/id situation as above.